hack/jenkins/job-configs/global.yaml

```diff
@@ -29,6 +29,33 @@
                     else
                       curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-to-gcs.sh" | bash -
                     fi
+                - shell: |
+                    curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-finished.sh" > upload-finished.sh
+                    chmod 777 upload-finished.sh
```

Comment:
`chmod +x` is probably all that we need.

also this is horrible hack and I love it. do we have the necessary plugin installed?